### PR TITLE
FEATURE: FET-236 Pull request description template

### PR DESCRIPTION
**Pull request by martinmosko**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3334/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2647/ : SUCCESS in 5m 35s

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- [ ] Verify code changes ([checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
+- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
+- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
+- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
+- [ ] Successful `extended test - examples`
+- [ ] Successful `extended test - storybook`
+- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)
+
+# Related PRs
+- [gdc-app-components](https://github.com/gooddata/gdc-app-components):
```
Comment (ivan-nejezchleb) [inline .github/pull_request_template.md]: not sure whether here we need links here when there will be link to individual PRs behind ':'

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+# PR Checklist
+
+- [ ] Verify code changes ([checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
+- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
+- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
+- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
+- [ ] Successful `extended test - examples`
+- [ ] Successful `extended test - storybook`
+- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)
```
Comment (ptquang86) [inline .github/pull_request_template.md]: How about [ ]  Squash all commit into one?

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+Command | Description
+--- | ---
+`extended test - examples` | Live examples tests
+`extended test - storybook` | Storybook screenshot tests
+
+See more [options](https://confluence.intgdc.com/display/~petr.dolejsi/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).
+
+# PR Checklist
+
+- [ ] Verify code changes ([checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
```
Comment (xMort) [inline .github/pull_request_template.md]: _C_hecklist?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3566/ : SUCCESS in 28s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2853/ : SUCCESS in 2m 31s

File: .github/pull_request_template.md (hunk 3 above)
Comment (martinmosko) [reply .github/pull_request_template.md]: Fixed

File: .github/pull_request_template.md (hunk 1 above)
Comment (martinmosko) [reply .github/pull_request_template.md]: Fixed

File: .github/pull_request_template.md (hunk 2 above)
Comment (martinmosko) [reply .github/pull_request_template.md]: this information is already part of `Check PR owner responsibilities` link mentioned in the template...

Do you feel that its important to you to have it here?

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3609/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2886/ : SUCCESS in 4m 39s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/790/ : SUCCESS in 21s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2894/ : SUCCESS in 7m 44s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1545/ : SUCCESS in 16m 29s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/780/ : SUCCESS in 20m 21s

**Pull request merged**